chromadb/config.py

```diff
@@ -116,6 +116,7 @@ class Settings(BaseSettings):  # type: ignore
     is_persistent: bool = False
     persist_directory: str = "./chroma"
 
+    chroma_memory_limit_bytes: int = 0
     chroma_server_host: Optional[str] = None
```

Comment:
Can we introduce a config - called segment_manager_cache_policy and make this one of many types?